README.md

```diff
@@ -125,37 +125,41 @@ The URL and credentials details are necessary to authenticate with the identity
 The credentials are a combination of "id" and "name" used to uniquely identify projects, users and their domains.
 When using only the name, a domain must be specified to guarantee a unique record from the Identity service.
 
-The following parameters can be used:  
-* `:domain_id`  
-  Domain id used for authentication scope  
-  Default: `"default"`  
-* `:domain`  
-  Domain name used for authentication scope  
-  Default: `"Default"`  
-* `:project_id`  
-  Project id  
-* `:project`  
-  Project name  
-* `:project_domain_id`  
+The following parameters can be used:
+* `:domain_id`
+  Domain id used for authentication scope
+  Default: `"default"`
+* `:domain`
+  Domain name used for authentication scope
+  Default: `"Default"`
+* `:project_id`
+  Project id
+* `:project`
+  Project name
+* `:project_domain_id`
   Project domain id
-* `:project_domain`  
-  Project domain name  
-* `:tenant_id`  
-  Tenant id, used only for Keystone v2.0  
-* `:tenant`  
-  Tenant name, used only for Keystone v2.0  
-* `:user_id`  
-  User id  
-* `:user`  
-  User name  
-* `:user_domain_id`  
+* `:project_domain`
+  Project domain name
+* `:tenant_id`
+  Tenant id, used only for Keystone v2.0
+* `:tenant`
+  Tenant name, used only for Keystone v2.0
+* `:user_id`
+  User id
+* `:user`
+  User name
+* `:user_domain_id`
   User domain id
-* `:user_domain`  
-  User domain name  
+* `:user_domain`
+  User domain name
 * `:password`
   Password for user. Cannot be used together with `:token`.
 * `:token`
   Previous Keystone v3 token for user. Can only be used with Keystone v3. Overrides all user and password parameters.
+* `:context`
+  I you have a proper context with `token`, `catalog` and `expire date` you can bypass the authentication to save the api call
+  Overrides all user and password parameters
+  Example: ``{:context => { :token => token, :catalog => catalog, :expires_at => expires_at }}``
```

Comment:
I guess you mean `expires`, should match https://github.com/sapcc/misty/blob/fa1d200173b1931602a52f752ed502ceb1744380/lib/misty/auth.rb#L38